src/views/Alteration.vue

```diff
@@ -13,11 +13,14 @@
       :personList="getPeopleAndRoles"
       :isSummary="true"
     />
-    <list-share-class
+
+    <share-structure
       class="mt-10"
       :shareClasses="getShareClasses"
-      :isSummary="true"
+      @setShareClass="setShareClasses($event)"
+      @haveChanges="shareStructChanges = $event"
```

Comment:
I believe we're getting away from setting data (or managing a ton of state data) in the views. The ShareStructure component can simply store its data (and whether it has changes).

Reply:
Same comment for the Correction view.

Have a look at `isFilingChanged()`, `isFilingValid()`, etc.

Reply:
ok, no prob. Will refactor a few things here, on `Correction` and in the `ShareStructure` component!

Reply:
done and done, refactored so the ShareStructure component handles state in both Correction and Alteration.

Reply:
Please delete obsolete getters and setters in Alteration.vue. Thx.

Reply:
You bet, removed them from Correction, i will double check Alteration.

Reply:
Fixed in next commit!